collector/mongod/metrics.go

```diff
@@ -397,7 +397,7 @@ type CursorStats struct {
 // Export exports the cursor stats.
 func (cursorStats *CursorStats) Export(ch chan<- prometheus.Metric) {
 	metricsCursorTimedOutTotal.Set(cursorStats.TimedOut)
-	metricsCursorOpen.WithLabelValues("noTimeout").Set(cursorStats.Open.NoTimeout)
+	metricsCursorOpen.WithLabelValues("timed_out").Set(cursorStats.Open.NoTimeout)
```

Comment:
I've reverted this back to 'noTimeout' to match the driver setting. To conclude: this metric is telling us the current number of connections who are using the "DBQuery.Option.noTimeout" option.